### PR TITLE
Integrating @tizentv/tools interface

### PR DESCRIPTION
**Pull request by keyuan0701**

### CODE REVIEW

**Review by xueli-hu — COMMENTED** [review]
good

**Review by chunhua-liu — DISMISSED** [review]

**Review by wb0716 — COMMENTED** [review]
Checked!

**Review by xujiangang-git — COMMENTED** [review]
checked

File: lib/appLauncher.js [hunk 1]
```diff
@@ -5,11 +5,8 @@ const { existsSync } = require('fs');
 const { get } = require('http');
```
Comment (prozanne) [inline lib/appLauncher.js:5]: I think downloadManager.js is also included in @tizentv/tools.
But in webIDE-common-tizentv, downloadManager.js is included as well, and even used for downloading certification resources.
Why can't it be united?

File: lib/appLauncher.js [hunk 2]
```diff
@@ -5,11 +5,8 @@ const { existsSync } = require('fs');
...
 const httpGet = get;
```
Comment (prozanne) [inline lib/appLauncher.js:6]: @tizentv/tools need to be added on package.json.

**Review by jianhua-git — COMMENTED** [review]
checked

File: lib/appLauncher.js (hunk 1 above)
Comment (keyuan0701) [reply lib/appLauncher.js:5]: done

File: lib/appLauncher.js (hunk 2 above)
Comment (keyuan0701) [reply lib/appLauncher.js:6]: done

File: lib/tizenCertificateManager.js [hunk 3]
```diff
@@ -6,15 +6,7 @@ const crypto = require('crypto');
 const cryptUtil = require('./cryptUtil');
```
Comment (prozanne) [inline lib/tizenCertificateManager.js:6]: I've realized something is different with made by Tizen Studio.

In webide-common-tizentv,
First, there is no parameter for Certificate profile name, while there is required in Tizen Studio. (I guess this value is a real name for profile which can be activated.)

For test, I put these same values for making a Tizen certification.
"certificate profile name" : certificateprofilename
"keyfilename": keyfilename
"authorname": authorname
"password": password

And then, the results were different.

`In profiles.xml made by Tizen Studio.`
```xml
<profiles active="certificateprofilename" version="3.1">
<profile name="certificateprofilename">
<profileitem ca="C:\tizen-studio\tools\certificate-generator\certificates\developer\tizen-developer-ca.cer" distributor="0" key="C:/tizen-studio-data/keystore/author/keyfilename.p12" password="C:\tizen-studio-data\keystore\author\keyfilename.pwd" rootca=""/>
<profileitem ca="C:\tizen-studio\tools\certificate-generator\certificates\distributor\sdk-partner\tizen-distributor-ca.cer" distributor="1" key="C:\tizen-studio\tools\certificate-generator\certificates\distributor\sdk-partner\tizen-distributor-signer.p12" password="C:\tizen-studio-data\tools\certificate-generator\certificates\distributor\sdk-partner\tizen-distributor-signer.pwd" rootca=""/>
<profileitem ca="" distributor="2" key="" password="" rootca=""/>
</profile>
```

`In profiles.xml made by webide-common-tizentv`
```xml
<profiles active="authorname" version="3.1">
  <profile name="authorname">
    <profileitem ca="E:\dev\Wits\node_modules\@tizentv\webide-common-tizentv\tools\certificate-generator\certificates\developer\tizen-developer-ca.cer" distributor="0" key="E:\dev\Wits\resource\Author\keyfilename.p12" password="E:\dev\Wits\resource\Author\keyfilename.pwd" rootca=""/>
    <profileitem ca="E:\dev\Wits\node_modules\@tizentv\webide-common-tizentv\tools\certificate-generator\certificates\distributor\sdk-partner\tizen-distributor-ca.cer" distributor="1" key="E:\dev\Wits\node_modules\@tizentv\webide-common-tizentv\tools\certificate-generator\certificates\distributor\sdk-partner\tizen-distributor-signer.p12" password="E:\dev\Wits\node_modules\@tizentv\webide-common-tizentv\tools\certificate-generator\certificates\distributor\sdk-partner\tizen-distributor-signer.pwd" rootca=""/>
    <profileitem ca="" distributor="2" key="" password="" rootca=""/>
  </profile>
```

The results should be same.
Please check this point.

**Review by prozanne — APPROVED** [review]
Cool

**Comment by capscrom** [comment]
LGTM

**Pull request merged**